### PR TITLE
Fix image upload

### PR DESCRIPTION
**Pull request by amoreira**
Change image upload endpoint and adds 'implements' to component descriptor.

### CODE REVIEW

File: src/editors/BannerEditor/ImageUpload.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ class ImageUpload extends React.Component {
   uploadImgSuccess = (fileName, response) => {
     if (response.status == 201) {
       this.setState({ uploading: false });
-      this.props.updateUrl(`http://vtex-fs-env-beta.elasticbeanstalk.com/${response.headers.location}`);
+      this.props.updateUrl(`http://vtexfs-env-beta.us-east-1.elasticbeanstalk.com/${response.headers.location}`);
```
Comment (tamorim) [inline src/editors/BannerEditor/ImageUpload.js]: @amoreira Why not use the resource?

File: storefront/components/Banner.json [hunk 2]
```diff
@@ -3,5 +3,5 @@
     "Banner.js",
     "editors/index.js"
   ],
-  "implements": ["HomePage@vtex.storefront-theme#banner"]
+  "implements": ["HomePage@vtex.storefront-theme#banner", "HomePage@vtex.storefront-theme#banner-1"]
```
Comment (tamorim) [inline storefront/components/Banner.json]: @amoreira This is super weird... the placeholder is basically the same, we can't make **banner** work for without this stuff?
If a page has 3 placeholders for banners the dev has to write that he implements every one of them? :/

File: package.json [hunk 3]
```diff
@@ -42,7 +42,8 @@
     "postreleasy": "vtex publish"
   },
   "dependencies": {
-    "vtex-css-utils": "^1.1.9",
-    "react-slick": "^0.9.2"
+    "json-loader": "^0.5.4",
+    "react-slick": "^0.9.2",
```
Comment (tamorim) [inline package.json]: @amoreira Update react-slick to the latest version https://github.com/akiran/react-slick/releases
It has some bugfixes!

File: src/services/Arquivos.js [hunk 4]
```diff
@@ -4,13 +4,17 @@ import { stores } from 'sdk';
 export default function uploadImage(file) {
   const accountName = stores.ContextStore.getState().get('accountName');
   let uploadImageResource = `/_resources/arquivos@vtex.banner/${accountName}/images/`;
-  let data = new FormData().append('image', file, file.name);
+
+  let data = new FormData();
+  data.append('image', file);
+
   let relativePath = uploadImageResource + file.name;
   let workspace = (`; ${document.cookie}`).split('; vtex_workspace=').pop().split(';').shift();
```
Comment (tamorim) [inline src/services/Arquivos.js]: @amoreira You can spare this cookie retrieval is you import the services from the sdk.
```javascript
import { services } from 'sdk';
...
// Gets the current workspace
const workspace = services.storefront.defaultHeaders['x-vtex-workspace'];
```

File: package.json (hunk 3 above)
Comment (tamorim) [inline package.json]: @amoreira Actually, forget this one, it's out of the scope of the PR!
When I merge this one I'll open a PR to solve that and remove that swipe bug on the desktop :)

**Comment by amoreira** [comment]
@tamorim done!

File: src/components/Banner/index.js [hunk 5]
```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { stores } from 'sdk';
```
Comment (tamorim) [inline src/components/Banner/index.js]: @amoreira The compiler warns: defined but never used.

File: src/services/Arquivos.js [hunk 6]
```diff
@@ -1,16 +1,21 @@
 import axios from 'axios';
 import { stores } from 'sdk';
 
+const workspace = services.storefront.defaultHeaders['x-vtex-workspace'];
```
Comment (tamorim) [inline src/services/Arquivos.js]: @amoreira Forgot the import for services...

**Pull request merged**